### PR TITLE
Added Personio to Readme

### PR DESCRIPTION
**Pull request by leisemann**
Added Personio (HR Tool) to Readme

### CODE REVIEW

**Comment by grayum** [comment]
We've added new columns to add other relevant CVE's. Please fetch, update accordingly and issue a new PR. Thank you.

File: software/README.md [hunk 1]
```diff
@@ -2394,6 +2394,7 @@ _Note: daily releases of this software list are listed, including CSV and JSON f
 | PDQ | PDQ Deploy | All versions | | Investigation | | | [source](https://www.pdq.com/blog/log4j-vulnerability-cve-2021-44228/?utm_content=190941012&utm_medium=social&utm_source=twitter&hss_channel=tw-90432152) |
 | PDQ | PDQ Inventory | All versions | | Investigation | | | [source](https://www.pdq.com/blog/log4j-vulnerability-cve-2021-44228/?utm_content=190941012&utm_medium=social&utm_source=twitter&hss_channel=tw-90432152) |
 | Pega | Pega Platform | 7.3.x - 8.6.x | Not vuln | Fix | | Hotfixes made available for registered customers by Pega. When using Stream nodes, the embedded Kafka instances require a separate hotfix to be installed. | [source](https://docs.pega.com/security-advisory/security-advisory-apache-log4j-zero-day-vulnerability) |
+| Personio | All | Unknown |  | Fix | Investigation | | [source](https://status.personio.de/incidents/kn4c6mf6lpdv) |
```
Comment (maertsen) [inline software/README.md:2397]:
```suggestion
| Personio | All | Unknown |  | Fix | Investigation | | | [source](https://status.personio.de/incidents/kn4c6mf6lpdv) |
```

If you have any additional information about the status of the other CVE, that'd be helpful.

**Review by maertsen — CHANGES_REQUESTED** [review]
@leisemann, we can merge when you apply the suggested change.

**Comment by maertsen** [comment]
thanks @leisemann

**Pull request merged**